zellij-server/src/ui/pane_boundaries_frame.rs

```diff
@@ -665,14 +668,16 @@ impl PaneFrame {
             } else {
                 self.empty_undertitle(max_undertitle_length)
             }
-        }
+        };
+        Ok(res)
     }
-    pub fn render(&self) -> (Vec<CharacterChunk>, Option<String>) {
+    pub fn render(&self) -> Result<(Vec<CharacterChunk>, Option<String>)> {
+        let err_context = || "failed to render";
```

Comment:
We have many functions called `render` in the code, so we can add some additional information:

```suggestion
        let err_context = || "failed to render pane frame";
```